### PR TITLE
Implement toggle button for enter/exit VR in Sumerian Scene Component

### PR DESCRIPTION
**Pull request by jordanranz**
*Issue #, if available:*
#2595 

*Description of changes:*
- Toggle enter/exit VR for UI components
- Exposed vrPresentationActive property on Sumerian scene controller object

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/xr/src/Providers/SumerianProvider.ts [hunk 1]
```diff
@@ -224,6 +224,11 @@ export class SumerianProvider extends AbstractXRProvider {
     return sceneController.vrCapable;
   }
 
+  public vrPresentationActive(sceneName: string): boolean {
+    const sceneController = this.getSceneController(sceneName);
```
Comment (elorzafe) [inline packages/xr/src/Providers/SumerianProvider.ts]: I think this needs to check if the sceneController is not `undefined` also for L224

Comment (jordanranz) [reply packages/xr/src/Providers/SumerianProvider.ts]: These checks actually exist and throw an error inside of the getSceneController method:
https://github.com/aws-amplify/amplify-js/blob/master/packages/xr/src/Providers/SumerianProvider.ts#L198

I think that covers this comment yea?

Comment (elorzafe) [reply packages/xr/src/Providers/SumerianProvider.ts]: Nice!, Just on thing I noticed, I would rename the method to `isVRPresentationActive`?

File: packages/xr/src/Providers/SumerianProvider.ts [hunk 2]
```diff
@@ -224,7 +224,7 @@ export class SumerianProvider extends AbstractXRProvider {
     return sceneController.vrCapable;
   }
 
-  public vrPresentationActive(sceneName: string): boolean {
+  public isVRPresentationActive(sceneName: string): boolean {
     const sceneController = this.getSceneController(sceneName);
     return sceneController.vrPresentationActive;
```
Comment (elorzafe) [inline packages/xr/src/Providers/SumerianProvider.ts]: This was what you mention, that Sumerian uses that name internally right? Just double checking ;)

Comment (jordanranz) [reply packages/xr/src/Providers/SumerianProvider.ts]: correct. We use their sceneController api.

File: packages/xr/src/Providers/SumerianProvider.ts (hunk 1 above)
Comment (jordanranz) [reply packages/xr/src/Providers/SumerianProvider.ts]: done!

**Pull request merged**